components/dialog/demo/dialog-confirm.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
```

Comment:
This is same demo as before, I just moved it into its own file since I was creating a new page for fullscreen dialog anyway.